lib/commands/test-init.ts

```diff
@@ -37,6 +37,25 @@ class TestInitCommand implements ICommand {
 				'save-dev': true,
 				optional: false,
 			});
+
+			let modulePath = path.join(projectDir, "node_modules", mod);
+			let modulePackageJsonPath = path.join(modulePath, "package.json");
+			let modulePackageJsonContent = this.$fs.readJson(modulePackageJsonPath);
+			let modulePeerDependencies = modulePackageJsonContent.peerDependencies || {};
+
+			for (let peerDependency in modulePeerDependencies) {
+				let dependencyVersion = modulePeerDependencies[peerDependency] || "*";
+
+				// catch errors when a peerDependency is already installed
+				// e.g karma is installed; karma-jasmine depends on karma and will try to install it again
+				try {
+					await this.$npm.install(`${peerDependency}@${dependencyVersion}`, projectDir, {
+						'save-dev': true
+					});
+				} catch (e) {
+					this.$logger.info(e.message);
```

Comment:
@Pip3r4o What i meant is try to run `npm install karma-chrome-launcher@1.x || ~0.2.2` and it will fail. Therefore we need quotes -> `npm install karma-chrome-launcher@"1.x || ~0.2.2"`. The above versions are pasted from https://github.com/karma-runner/karma-jasmine/blob/master/package.json so they are valid.